src/app/routes/routeProjects.ts

```diff
@@ -8,65 +8,100 @@ import { verifyAccessToken } from "../middlewares/verifyAccessToken";
 import { BUILD_MESSAGE } from "../../config/constants";
 import { ProjectService } from "../../domains/projectService";
 import { UserService } from "../../domains/userService";
-import { container } from "../../domains/@config/di.config";
+import { container } from "../../config/di.config";
 import { handleAsync } from "../utils/handleAsync";
 import { Logger as log } from "../../utils/Logger";
+import Config from "../../config";
+
+import type { TokenClient } from "../../config/di.config";
+
+const project = z.object({
+  userId: z.string(),
+  space: z.string(),
+  repoName: z.string(),
+  repoCloneUrl: z.string(),
+  projectName: z.string(),
+  projectUpdatedAt: z.string(),
+  framework: z.union([
```

Comment:
```suggestion
  framework: z.union(keys(CLIENT_FRAMEWORK_INFO).map(framwork => z.literal(framwork))),
```
이렇게는 어떨까요? (`keys`는 lodash 메서드입니다..!)

Reply:
좋네요..! 반영하겠습다아~